internal/sdkv2provider/resource_cloudflare_device_settings_policy.go

```diff
@@ -269,7 +282,9 @@ func buildDeviceSettingsPolicyRequest(d *schema.ResourceData) (cloudflare.Device
 			Mode: cloudflare.ServiceMode(d.Get("service_mode_v2_mode").(string)),
 			Port: d.Get("service_mode_v2_port").(int),
 		},
-		ExcludeOfficeIps: cloudflare.BoolPtr(d.Get("exclude_office_ips").(bool)),
+		ExcludeOfficeIps:   cloudflare.BoolPtr(d.Get("exclude_office_ips").(bool)),
+		LANAllowMinutes:    cloudflare.UintPtr(d.Get("lan_allow_minutes").(uint)),
```

Comment:
@da-cf i don't think this will work. the casting here should be `(int)` (as that is what the schema has) so we'll need to convert it to `uint`.

Reply:
I'm not sure if I follow, but I made the changes to the best of my understanding. Let me know if I need to implement this differently.